### PR TITLE
Create code-review-dance-activity.md

### PR DESCRIPTION
**Pull request by blakeLarkin**
This is my new version of the code review activity, based on the existing Github Tutorial.

### CODE REVIEW

File: code-review-dance-activity.md
Comment (ykharitonova) [inline code-review-dance-activity.md]: what does it mean: "with mandatory code reviews"?

File: code-review-dance-activity.md
Comment (ykharitonova) [inline code-review-dance-activity.md]: Set's up --> Sets up

File: code-review-dance-activity.md
Comment (ykharitonova) [inline code-review-dance-activity.md]: no need for an apostrophe

File: code-review-dance-activity.md
Comment (ykharitonova) [inline code-review-dance-activity.md]: fosters

File: code-review-dance-activity.md
Comment (ykharitonova) [reply code-review-dance-activity.md]: Is it what you explained in Step 1? If they do it there, then it's not a prereq?

File: code-review-dance-activity.md
Comment (ykharitonova) [inline code-review-dance-activity.md]: use code marks, i.e. `backticks`, for code-related lines

File: code-review-dance-activity.md
Comment (ykharitonova) [inline code-review-dance-activity.md]: the "Pull Requests" tab on GitHub

File: code-review-dance-activity.md
Comment (ykharitonova) [inline code-review-dance-activity.md]: Check your "Pull Requests" tab on GitHub

File: code-review-dance-activity.md
Comment (ykharitonova) [inline code-review-dance-activity.md]: Isn't this dependent on the configuration of the repo? I.e., is it enough to just be the collaborator on the repo to merge the commit in?

**Comment by blakeLarkin** [comment]
Need to explain how to make a PR and make it explicit to use the repo from the other exercise

File: code-review-dance-activity.md
Comment (blakeLarkin) [inline code-review-dance-activity.md]: Clarify what is meant by "project's 'Settings'"

File: code-review-dance-activity.md
Comment (blakeLarkin) [inline code-review-dance-activity.md]: This is where I may need to describe making PRs

File: code-review-dance-activity.md
Comment (blakeLarkin) [inline code-review-dance-activity.md]: Say how to leave comments

Comment (blakeLarkin) [reply code-review-dance-activity.md]: explain difference between single comments and a review

Comment (blakeLarkin) [reply code-review-dance-activity.md]: and if doing a review make sure that you don't approve it if there are issues

Comment (blakeLarkin) [reply code-review-dance-activity.md]: Infact, select "Request Changes"

File: code-review-dance-activity.md
Comment (blakeLarkin) [inline code-review-dance-activity.md]: leave good commit messages

File: code-review-dance-activity.md
Comment (blakeLarkin) [inline code-review-dance-activity.md:117]: Talk about deleting branches

**Review by ykharitonova — CHANGES_REQUESTED** [review]
Looks like my comments are currently "pending"; let's see if "request changes" puts them through...

File: code-review-dance-activity.md
Comment (blakeLarkin) [reply code-review-dance-activity.md]: Yes, I changed the instructions earlier to allow collaborators to approve PRs (not only owners). I believe if they are in the same groups as the first assignment it should work.

File: code-review-dance-activity.md
Comment (blakeLarkin) [reply code-review-dance-activity.md:117]: I added this

**Review by ykharitonova — APPROVED** [review]

**Review by azwildcat — APPROVED** [review]
Let's merge this activity into the main repo.

**Review by azwildcat — APPROVED** [review]

**Pull request merged**
